pytest_parallel/__init__.py

```diff
@@ -338,10 +349,22 @@ def pytest_runtestloop(self, session):
         processes = []
         for _ in range(self.workers):
             process = Process(target=process_with_threads,
-                              args=(queue, session, tests_per_worker))
+                              args=(queue, session, tests_per_worker, errors))
             process.start()
             processes.append(process)
         [p.join() for p in processes]
         queue.join()
 
+        if not errors.empty():
+            import six
+            import pickle
+
+            exc = RuntimeError("pytest-parallel got {} errors, raising the first.".format(
+                errors.qsize()
+            ))
+            err = pickle.loads(errors.get())
+            err[1].__traceback__ = err[2]
+
+            six.raise_from(exc, err[1])
```

Comment:
Wondered about your opinion about py2 support here, but since six is used already I've thought to stick with it.

Reply:
six is a relic from my attempt at supporting py2. Consistency works for me 👍

Reply:
six is a relic from my attempt at supporting py2. Consistency works for me 👍